src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/FileVersionInfoTest.cs

```diff
@@ -102,6 +102,17 @@ public void FileVersionInfo_NonExistentFile_FileNotFound()
                 FileVersionInfo.GetVersionInfo(Path.Combine(Directory.GetCurrentDirectory(), TestNotFoundFileName)));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "Don't want to create temp file in app container current directory")]
+        public void FileVersionInfo_RelativePath_CorrectFilePath()
+        {
+            using (new FileStream("kernelbase.dll", FileMode.CreateNew, FileAccess.ReadWrite, FileShare.None, 0x1000, FileOptions.DeleteOnClose))
+            {
+                FileVersionInfo fvi = FileVersionInfo.GetVersionInfo("kernelbase.dll");
```

Comment:
Does this test fail without the fix? I think this test will pass both before and after the fix.